packages/material-ui/src/LinearProgress/LinearProgress.js

```diff
@@ -211,7 +211,7 @@ const LinearProgress = React.forwardRef(function LinearProgress(props, ref) {
   if (variant === 'determinate' || variant === 'buffer') {
     if (value !== undefined) {
       rootProps['aria-valuenow'] = Math.round(value);
-      inlineStyles.bar1.transform = `scaleX(${value / 100})`;
+      inlineStyles.bar1.width = `${value}%`;
```

Comment:
As far as I know this slow. We opted for transform in the `Slider` for those exact reasons. Could you look into other solutions for progress bars? I suspect those problems are already solved somewhere else. Maybe use `translate` instead of `scale`?

Reply:
- https://csstriggers.com/transform
- https://csstriggers.com/width

Reply:
The monimal case is to display a single linear progress on your page. I doubt it will make a difference.

Reply:
I already implemented the changes and it still works as expected so I do agree with @eps1lon here.

Reply:
Great 👍

Reply:
It's solving a bug too 😍: #15063.

Reply:
I knew it 🤓 (it's a joke 😆)